File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/plagiarized/L1/01/L1.kt
package IR

import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`)

    print("Enter the radius and length of a cylinder: ")
    val radius = input.nextDouble()
    val length = input.nextDouble()
    val area = radius * radius * 3.14159
    val volume = area * length
    println("The area is $area, ")
    print("The volume of the cylinder is $volume")
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L1/01/L1.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter the radius and length of a cylinder: ', (input) => {
  const [radiusStr, lengthStr] = input.split(' ');
  const radius = parseFloat(radiusStr);
  const length = parseFloat(lengthStr);
  const area = radius * radius * 3.14159;
  const volume = area * length;
  console.log(`The area is ${area}, `);
  console.log(`The volume of the cylinder is ${volume}`);
  rl.close();
});

